modules/getting-started-web-console-creating-new-project.adoc

```diff
@@ -1,33 +1,25 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-web-console.adoc
+// * tutorials/dev-app-web-console.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-web-console-creating-new-project_{context}"]
-= Creating a new project
+= Creating a project
 
-A project enables a community of users to organize and manage their content in isolation. Projects are {product-title} extensions to Kubernetes namespaces. Projects have additional features that enable user self-provisioning.
+A _project_ enables a community of users to organize and manage their content in isolation. Projects are {product-title} extensions to Kubernetes namespaces. Projects have additional features that enable user self-provisioning. Each project has its own set of objects, policies, constraints, and service accounts.
 
-Users must receive access to projects from administrators. Cluster administrators can allow developers to create their own projects. In most cases, users automatically have access to their own projects.
+Cluster administrators can allow developers to create their own projects. In most cases, you automatically have access to your own projects. Administrators can grant access to other projects as needed.
 
-Each project has its own set of objects, policies, constraints, and service accounts.
+This procedure creates a new project called `user-getting-started`.  You will use this project throughout the rest of this tutorial.
 
 .Prerequisites
 
-* You are logged in to the {product-title} web console.
-* You are in the *Developer* perspective.
 * You have the appropriate roles and permissions in a project to create applications and other workloads in {product-title}.
+// TODO: This requirement isn't in the CLI version. Is it necessary?
```

Comment:
@AlexonOliveiraRH I think we can remove the above line ("You have the appropriate roles..."). It's not in the CLI version. wdyt?

Reply:
> @AlexonOliveiraRH I think we can remove the above line ("You have the appropriate roles..."). It's not in the CLI version. wdyt?

@bergerhoffer although this is not CLI, permissions are bound across all clusters resources, independently of which platform you're using, i.e. CLI or web console, the user still needs to guarantee the proper permissions.

Reply:
@AlexonOliveiraRH What permissions should we say that they need to have then? (And if we add it here, we should also add that to the CLI version)

Reply:
Hm, this still doesn't seem to check out to me, because if they are the ones that created the project, then shouldn't they automatically get whatever permissions needed to work in that project?

The only thing I could see saying here is about whether someone has access to the cluster, but doesn't have permission to create a project (if there is such a role like that). wdyt?

Reply:
> Hm, this still doesn't seem to check out to me, because if they are the ones that created the project, then shouldn't they automatically get whatever permissions needed to work in that project?
> 
> The only thing I could see saying here is about whether someone has access to the cluster, but doesn't have permission to create a project (if there is such a role like that). wdyt?

@bergerhoffer for example, the same user could get only a `view` permission, so this user would be able to navigate into the project, see things, but wouldn't be able to create nothing, neither change anything. That's why the cluster admin should grant this user at least `edit` permission (or `admin` in the namespace scope only) to create the project and do the rest of the tutorial